src/Infrastructure/Service/ErrorCommandListener.php

```diff
@@ -9,33 +9,45 @@
 
 use Doctrine\DBAL\Exception\TableNotFoundException;
 use SprykerSdk\Sdk\Core\Application\Dependency\ErrorCommandListenerInterface;
-use SprykerSdk\Sdk\Core\Application\Exception\ProjectWorkflowException;
 use SprykerSdk\Sdk\Core\Application\Exception\SettingsNotInitializedException;
+use SprykerSdk\Sdk\Presentation\Console\Command\InitSdkCommand;
+use SprykerSdk\Sdk\Presentation\Console\Command\UpdateSdkCommand;
 use Symfony\Component\Console\Event\ConsoleErrorEvent;
 
 class ErrorCommandListener implements ErrorCommandListenerInterface
 {
+    /**
+     * @var bool
+     */
+    protected bool $isDebug;
+
+    /**
+     * @param bool $isDebug
+     */
+    public function __construct(bool $isDebug)
+    {
+        $this->isDebug = $isDebug;
+    }
+
     /**
      * @param \Symfony\Component\Console\Event\ConsoleErrorEvent $event
      *
      * @return void
      */
     public function handle(ConsoleErrorEvent $event): void
     {
-        if (
-            !($event->getError() instanceof TableNotFoundException || $event->getError() instanceof SettingsNotInitializedException)
-            || $event->getOutput()->isDebug()
-        ) {
+        if ($this->isDebug && $event->getOutput()->isDebug()) {
             return;
         }
 
-        $event->setError(
-            new ProjectWorkflowException(
-                'You need to init or update the SDK. you need to run \'sdk:init:sdk\' or \'sdk:update:all\' command.',
-                0,
-                $event->getError(),
-            ),
-        );
         $event->setExitCode(0);
+
+        if ($event->getError() instanceof TableNotFoundException || $event->getError() instanceof SettingsNotInitializedException) {
+            $event->getOutput()->writeln(sprintf('<error>You need to init or update the SDK. you need to run \'%s\' or \'%s\' command.</error>', InitSdkCommand::NAME, UpdateSdkCommand::NAME));
```

Comment:
What if we decorate Symfony command output with additional functions like `writeErrorLn()`, `writeWarningLn()`, etc.?

Reply:
I'm not sure that we need to invest time to it now